plugins/org.eclipse.elk.core.meta/src/org/eclipse/elk/core/meta/formatting2/MetaDataFormatter.xtend

```diff
@@ -0,0 +1,173 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Kiel University and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.elk.core.meta.formatting2
+
+import org.eclipse.xtext.formatting2.AbstractFormatter2
+import org.eclipse.xtext.formatting2.IFormattableDocument
+import org.eclipse.elk.core.meta.metaData.MdBundle
+import org.eclipse.xtext.xbase.lib.Procedures.Procedure1
+import org.eclipse.xtext.formatting2.IHiddenRegionFormatter
+import org.eclipse.elk.core.meta.services.MetaDataGrammarAccess
+import org.eclipse.xtext.formatting2.regionaccess.ISemanticRegion
+import com.google.inject.Inject
+import org.eclipse.elk.core.meta.metaData.MdGroup
+import org.eclipse.elk.core.meta.metaData.MdOption
+import org.eclipse.elk.core.meta.metaData.MdOptionDependency
+import org.eclipse.elk.core.meta.metaData.MdAlgorithm
+import org.eclipse.elk.core.meta.metaData.MdCategory
+import org.eclipse.elk.core.meta.metaData.MdOptionSupport
+import org.eclipse.elk.core.meta.metaData.MdBundleMember
+import org.eclipse.elk.core.meta.metaData.MdGroupOrOption
+
+class MetaDataFormatter extends AbstractFormatter2 {
+
+    static val Procedure1<? super IHiddenRegionFormatter> one_space = [oneSpace]
+    static val Procedure1<? super IHiddenRegionFormatter> new_line = [newLine]
+    static val Procedure1<? super IHiddenRegionFormatter> indention = [indent]
+
+    @Inject extension MetaDataGrammarAccess
+    
+    def dispatch void format (MdBundleMember bundleMember, extension IFormattableDocument document){      
+        if(bundleMember instanceof MdAlgorithm){
```

Comment:
Check your formatting of xtend files please. There should be a space after `if` and one before `{` and so on.